src/components/molecules/news-and-events-title/news-and-events-title.html

```diff
@@ -0,0 +1,4 @@
+<div class="news-event-title">
+    <h2 class="news-title">News</h2>
+    <span class="news-subtitle">お知らせ</span>
+</div>
```

Comment:
```suggestion
<div class="news-events-title">
    <h2 class="main">News</h2>
    <span class="sub">お知らせ</span>
</div>
```
このパーツはorganisms/newsとorgasnisms/eventsで使いまわすパーツなので、news-titleというクラス名だとその点が微妙かもしれないです！

上のようなクラス名にするのは一つの案ですし、h2とspanにはクラスを付与せずに要素セレクタで記述してもいいと思います。